src/python/WMCore/WorkQueue/DataStructs/WorkQueueElement.py

```diff
@@ -101,13 +101,13 @@ def __init__(self, **kwargs):
         # is new data being added to the inputs i.e. open block with new files or dataset with new closed blocks?
         self.setdefault('OpenForNewData', False)
         # When was the last time we found new data (not the same as when new data was split), e.g. An open block was found
-        self.setdefault('TimestampFoundNewData', 0)
+        self.setdefault('TimestampFoundNewData', int(time.time()))
```

Comment:
From the change it is unclear what is convention about stored timestamps, do you intend to store local time or stick to UTC. If former, the change is fine, but if you'd like to have common time zone for all deployed instances then UTC will be much better choice.

Reply:
This is EPOCH time, so it should be a reference to "January 1, 1970, 00:00:00 (UTC)". I can add a comment in the line above though. Same reply for the question.